### ISSUE TITLE
RFE: Address mcpd "already exists" messages

### ISSUE THREAD

**Issue by walterk82** [F5Networks/f5-appsvcs-extension#122]
##### Do you already have an issue opened with F5 support?
Yes - C3417593 

##### ISSUE TYPE
 - Feature Idea

##### AS3 BUILD/ VERSION

```
3.13.0.3
```

##### BIGIP VERSION

```
14.1.0.6
```

##### SUMMARY

```
When posting AS3 declarations mcpd throws an `err` level message which causes false configuration error messages to fire

mcpd[526]: 01020066:3: The requested value list (/Common/appsvcs/dataStore) already exists in partition Common.
mcpd[526]: 01020037:3: The requested folder (/Common/appsvcs) already exists.
```

##### STEPS TO REPRODUCE

```

```

##### EXPECTED RESULTS

```
mcpd should not produce an err message if AS3 is working as designed.  This makes troubleshooting difficult and waste additional time from support.
```

##### ACTUAL RESULTS

```

```

**Comment by walterk82** [comment]
I have opened customer support case C3417593 
Thank-you.

**Comment by walterk82** [comment]
assigned BZ ID 961277

**Comment by dstokesf5** [comment]
This issue has been resolved as part of the 3.25.0 release.
